onionshare/onion.py

```diff
@@ -552,8 +555,11 @@ def connect(
         # Do the versions of stem and tor that I'm using support stealth onion services?
         try:
             res = self.c.create_ephemeral_hidden_service(
-                {1: 1}, basic_auth={"onionshare": None}, await_publication=False,
-                key_type="NEW",key_content="RSA1024"
+                {1: 1},
+                basic_auth={"onionshare": None},
+                await_publication=False,
+                key_type="NEW",
+                key_content="RSA1024",
             )
```

Comment:
@micahflee I think this has snuck in from an unrelated PR, but otherwise looks fine to me

Reply:
I believe that was just `black` reformatting the python code in `onion.py` on save. I had already merged in your PR that added that code.